scalafix/output-0_14/src/main/scala/fix/v0_14_0/FixAvroCoder16.scala

```diff
@@ -0,0 +1,11 @@
+package fix.v0_14_0
+
+import com.spotify.scio.values.SCollection
+import com.spotify.scio.avro._
```

Comment:
why do we add the import here ? It is a false positive ?

Reply:
It is needed in this case -- without it, `data.map(r => ("foo", r))` throws a Compilation error:

```
[error] /Users/clairem/scio/scalafix/output-0_14/src/main/scala/fix/v0_14_0/FixAvroCoder16.scala:9:13: 
[error] Cannot find an implicit Coder instance for type:
[error] 
[error]   >> (String, fix.v0_14_0.A)
```

Reply:
I forgot `A` is not a generic type but a `SpecificRecord` :man_facepalming:

Reply:
haha, we probably could have given it a better class name 😅